src/components/Header/Header.tsx

```diff
@@ -0,0 +1,22 @@
+import React from 'react'
+import mainIcon from '../../assets/icons/mainIcon.svg'
+import searchIcon from '../../assets/icons/searchIcon.svg'
+import personIcon from '../../assets/icons/personIcon.svg'
+import shoppingBagIcon from '../../assets/icons/shoppingBagIcon.svg'
+import './Header.scss'
+
+function Header() {
+  return (
+    <div className="container">
+        <button className="hamburger-button"/>
+        <img src={mainIcon} alt='main-logo' className="main-icon"/>
+        <div className="icons-container">
+            <img src={searchIcon} alt='shopping-bag' className="logo-container"/>
+            <img src={personIcon} alt='shopping-bag' className="logo-container"/>
+            <img src={shoppingBagIcon} alt='shopping-bag' className="logo-container"/>
+        </div>
```

Comment:
even tho we wont add functionality to the header icons, lets work as if they were functional.
meaning - instead of wrapping them within a div, a better practice would be making a list and each link/button will be a list item wrapping the image.

something like this:
```
<ul className="icons-container">
  <li>
     <img src={searchIcon} alt='shopping-bag' className="logo-container"/>
</li>
<li>
     <img src={personIcon} alt='shopping-bag' className="logo-container"/>
</li>
<li>
    <img src={shoppingBagIcon} alt='shopping-bag' className="logo-container"/
</li>
 </ul>
```